lms/djangoapps/course_api/views.py

```diff
@@ -280,18 +280,22 @@ def get_queryset(self):
         form = CourseListGetForm(self.request.query_params, initial={'requesting_user': self.request.user})
         if not form.is_valid():
             raise ValidationError(form.errors)
+        from openedx.core.djangoapps.util.forms import to_bool
+        include_ended = to_bool(self.request.GET.get('include_ended'))
         spec_courses = self.request.GET.get('spec_courses', 'false') in ['true', 'True']
         if spec_courses:
             return get_all_courses(
                 org=form.cleaned_data['org'],
                 filter_=form.cleaned_data['filter_'],
+                exclude_ended=not include_ended,
```

Comment:
You could refactor this and move the not to the variable definition and call it exclude_ended